finetune/config.py

```diff
@@ -54,7 +54,7 @@ def all_gpus(visible_gpus=None):
         device_ids = nvidia_device_ids()
         mapping = None
         # restricting GPUs based on env vars
-        if cuda_visible_devices:
+        if cuda_visible_devices is not None:
```

Comment:
I think this issue caused some strange warnings in prod?  We should actually prioritize this PR if for no other reason than this line.